x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -67,6 +65,51 @@ role descriptor with no privileges. The derived API key can be used for
 authentication; it will not have authority to call {es} APIs.
 
 --
++
+`applications`::: (list) A list of application privilege entries.
+`application` (required):::: (string) The name of the application to which this entry applies
+`privileges`:::: (list) A list of strings, where each element is the name of an application
+privilege or action.
+`resources`:::: (list) A list resources to which the privileges are applied.
```

Comment:
This is a problem with the eixsting page: Both `privileges` and `resources` are required. Since we are copying it over, might as well fix it here.